lib/animina_web/components/beta_registration_components.ex

```diff
@@ -319,28 +319,32 @@ defmodule AniminaWeb.BetaRegistrationComponents do
     ~H"""
     <div>
       <label
-        for="form_minimum_partner_age"
+        for="form_minimum_partner_height"
         class="block text-sm font-medium leading-6 text-gray-900 dark:text-white"
       >
         <%= with_locale(@language, fn -> %>
-          <%= gettext("Minimum age") %>
+          <%= gettext("Minimum height") %>
         <% end) %>
       </label>
-      <div phx-feedback-for={@f[:minimum_partner_age].name} class="mt-2">
-        <%= select(@f, :minimum_partner_age, Enum.map(18..110, &{&1, &1}),
-          prompt: with_locale(@language, fn -> gettext("doesn't matter") end),
-          value: @f[:minimum_partner_age].value,
+      <div phx-feedback-for={@f[:minimum_partner_height].name} class="mt-2">
+        <%= select(
+          @f,
+          :minimum_partner_height,
+          [{with_locale(@language, fn -> gettext("doesn't matter") end), nil}] ++
+            Enum.map(140..210, &{"#{&1} cm", &1}),
           class:
-            "block w-full rounded-md border-0 py-1.5 text-gray-900 dark:bg-gray-700 dark:text-white shadow-sm ring-1 ring-inset focus:ring-2 focus:ring-inset phx-no-feedback:ring-gray-300 phx-no-feedback:focus:ring-indigo-600 sm:text-sm sm:leading-6 " <>
-              unless(get_field_errors(@f[:minimum_partner_age], :minimum_partner_age) == [],
+            "block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset focus:ring-2 dark:bg-gray-700 dark:text-white focus:ring-inset phx-no-feedback:ring-gray-300 phx-no-feedback:focus:ring-indigo-600 sm:text-sm sm:leading-6 " <>
+              unless(
+                get_field_errors(@f[:minimum_partner_height], :minimum_partner_height) == [],
                 do: "ring-red-600 focus:ring-red-600",
                 else: "ring-gray-300 focus:ring-indigo-600"
-              )
+              ),
+          autofocus: true
         ) %>
 
-        <.error :for={msg <- get_field_errors(@f[:minimum_partner_age], :minimum_partner_age)}>
+        <.error :for={msg <- get_field_errors(@f[:minimum_partner_height], :minimum_partner_height)}>
           <%= with_locale(@language, fn -> %>
-            <%= gettext("Minimum age") <> " " <> msg %>
+            <%= gettext("Minimum height") <> " " <> msg %>
```

Comment:
Please ALWAYS(!) use something like `gettext("Hello, %{name}!", name: "Meg")` for this kind of thing. All languages are different. There is no way this order will not change for all languages.